core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -115,6 +117,17 @@ public boolean isValid() {
         return getProjectConfig() != null;
     }
 
+    @Override
+    public void close() throws IOException {
+        if (this.eventHandler instanceof Closeable) {
```

Comment:
Nit, but I tend to only prefix with `this.` when it's necessary. Like resolving variable name conflicts.